packages/web-react/src/components/SplitButton/README.md

```diff
@@ -115,6 +115,63 @@ On top of the API options, the components accept [additional attributes][readme-
 If you need more control over the styling of a component, you can use [style props][readme-style-props]
 and [escape hatches][readme-escape-hatches].
 
+## Uncontrolled Split Button
+
+Uncontrolled Split Button is combination of Button component and Dropdown component.
+It is used when you want to have a button with additional actions in a dropdown menu.
```

Comment:
Uncontrolled Split Button is combination of Button component and Dropdown component.
It is used when you want to have a button with additional actions in a dropdown menu.